package Exception;

public class TestLogin {
	public static void main(String[] args) throws LoginException {
		String name ="rays";
		if (name.equals("rays")) {
			System.out.println("valid user");
		}
		else {
			LoginException e=new LoginException();
			try {
				
			} catch (Exception e2) {
				System.out.println(e2);
				
			}
				
			} {
		}
		
	}

}
